### PR TITLE
add: translations for My Journal and My Time

### LINKED ISSUE #1475: Translate MyTime and MyJournal
**Is your feature request related to a problem? Please describe.**
It's very nice when you can use the app in your own language.

**Describe the solution you'd like**
Translate the header for MyTime and MyJournal.

**Describe alternatives you've considered**
None.

**Additional context**
Bottom menu is already translated. Please use the same wording.

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
resolves #1475 @tijlleenders

### CODE REVIEW

**Review by tijlleenders — CHANGES_REQUESTED** [review]
Thanks @vinaybadgujar102 

Please mind that, to my knowledge, all languages except German don't capitalize nouns. So 'My Time' should actually be 'My time'.  

I also see the camelCase for translations is not consistent, so made #1499 for that.

**Review by tijlleenders — CHANGES_REQUESTED** [review]
Can you also do it for the other pages in all languages (My feelings and My journal)?

**Comment by vinaybadgujar102** [comment]
I should keep the German translation as it is, right?

**Comment by tijlleenders** [comment]
Yes, German has the Rule that all Nouns are capitalized. I'm sure there's other languages like that, but I don't think they're in the currently supported list.

**Review by tijlleenders — APPROVED** [review]
Great!

File: src/pages/MyTimePage/MyTimePage.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ export const MyTimePage = () => {
   };
 
   return (
-    <AppLayout title="My Time">
+    <AppLayout title="mytime">
```
[the remark this answers is not in the document]
Comment (tijlleenders) [reply src/pages/MyTimePage/MyTimePage.tsx:79]: "My time" is correct.

[the remark this answers is not in the document]
Comment (tijlleenders) [reply src/pages/MyTimePage/MyTimePage.tsx:79]: The camelCase in the translation code is for a different issue.

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**